### DISCUSSION TITLE
Organisation, Channel and Framework Understanding

### DISCUSSION THREAD

**Discussion by coolbung** [Sunbird-Ed/Community#317] [Documentation]
We've shot this video as internal training for understanding organisation, channel and frameworks. Would like some of the key Sunbird devs to validate this understanding so we could continue creating more content on related topics. 

https://www.youtube.com/watch?v=wdTiO9FxKiA 

@Krishnaj20 @vrayulu @vinukumar-vs

**Comment by chitranshu-keshav** [comment]
@vinukumar-vs Can you please get the video validated and get back on this thread

cc: @Krishnaj20
